static/templates/user_profile_modal.hbs

```diff
@@ -63,5 +63,15 @@
             </div>
             {{/each}}
         </div>
+        <div id="total-list-sections">
+            <div id="groups-list-section" class="list-section">
+                <h4>Groups Part of</h4>
```

Comment:
h4 looks too big, we should make it look similar other heading in the modal (like GitHub, Mentor).

Instead of `Groups Part of`, just `User groups` would do.

Also, the commit message should be like : user_profile_modal: Show the `user groups` user belongs to.

Reply:
Please review. I have updated the css.